### DISCUSSION TITLE
just bought on the microsoft store

### DISCUSSION THREAD

**Discussion by smjLSH** [mifi/lossless-cut#1696] [General]
hi there- I heard about this from this video: https://www.youtube.com/watch?v=YLtGscod3zk
and just bought on the microsoft store because I am an idiot boomer and couldn't figure out the github "assets" stuff.  Happy to support- wish me luck!!

**Comment by mifi** [comment]
Thanks, hope you like it and good luck!

**Comment by smjLSH** [comment]
I am literally using it right now it is PERFECT!!  So easy- EXACTLY what I needed.  Thank you SOOOO much!!  Have a great day 😊😊😊

**Comment by smjLSH** [comment]
Hi Mikal-
Quick question- I noticed that the exported segments starting points seem to be different from where I selected them in Losslesscut.  Is there a trick to making the segments export more accurately?
Thanks!
-sheila

Ps I’m a video editing noob

**Reply by mifi** [reply]
Hi. This is limitation with lossless cutting, cuts will be aligned to the nearest previous keyframe. See [Known issues, limitations, troubleshooting, FAQ](https://github.com/mifi/lossless-cut/blob/master/issues.md)
alternatively you can try the experimental "smart cut" feature.
